src/Components/BottomText/BottomText.module.css

```diff
@@ -0,0 +1,24 @@
+.title{
+color: #2e66f5;
+font-size: 24px;
+margin-bottom: 20px;
+}
+.container{
+    display:flex;
+    flex-direction: column;
+    align-items: center;
+}
+.first_para{
+    padding: 10px;
+    margin-bottom:26px;
```

Comment:
this much margin is not feasible for layouting, make the layout proper using flex